src/helpers/logger.js

```diff
@@ -0,0 +1,33 @@
+// @ts-check
+
+/**
+ *
+ * @param {string} message
+ * @param {Object} options
+ * @param {string} options.level
+ * @param {Object} options.args
+ * @returns void
+ */
+function log (message, options) {
+  if (!options || !options.level) {
+    return console.log(message)
+  }
+  switch (options.level) {
+    case 'debug':
```

Comment:
would prefer these to be constants

Reply:
As would I. I don't know how to do enums in JS though.

Reply:
I have made an attempt. Can you please take a look, @thomasrockhu ?